CodeU_Assignement_3/src/codeu_assignement_3/SimpleDictionary.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */

```

Comment:
sure, np.
Is there a way to move it for good? I.e., when a new class created, it will not add it automatically?